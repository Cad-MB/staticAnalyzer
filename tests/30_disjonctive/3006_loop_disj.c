{
    int x;
    x = 0;
    while (x < 10) {
      if (x == 5) {
        x = x + 2;
      } else {
        x = x + 1;
      }
    }
    print_all;
  }
  