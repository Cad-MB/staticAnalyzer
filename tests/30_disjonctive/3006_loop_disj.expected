tests/30_disjunctive/3006_loop_disj.c:11.2-12: [ x in {[10;10]} ]
analysis ended